pybigquery/sqlalchemy_bigquery.py

```diff
@@ -966,3 +966,14 @@ def get_view_definition(self, connection, view_name, schema=None, **kw):
             view_name = f"{self.dataset_id}.{view_name}"
         view = client.get_table(view_name)
         return view.view_query
+
+
+try:
+    import alembic  # noqa
+except ImportError:
+    pass
+else:
+    from alembic.ddl import impl
```

Comment:
FWIW you could appease flake8 by just importing `impl` above, rather than the bare `import alembic`.